src/components/widgets/timelapse/TimelapseSettingsCard.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <collapsable-card
+    :title="$t('app.timelapse.title.timelapse_settings')"
+    icon="$cog"
+    class=""
+    :draggable="false"
+  >
+    <app-setting
+      :title="$t('app.timelapse.setting.enable')"
+      :sub-title="subtitleIfBlocked(enabledBlocked)"
+      :r-cols="2"
+    >
+      <v-switch
+        v-model="enabled"
+        hide-details
+        :disabled="enabledBlocked"
+        @click.native.stop
+      />
+    </app-setting>
+    <app-setting
+      :title="$t('app.timelapse.setting.auto_render')"
+      :sub-title="subtitleIfBlocked(autoRenderBlocked)"
+      :r-cols="2"
+    >
+      <v-switch
+        v-model="autoRender"
+        hide-details
+        :disabled="autoRenderBlocked"
+        @click.native.stop
+      />
+    </app-setting>
+    <div v-if="!frameCount">
```

Comment:
What do you think if we just disable the button instead of hiding this whole thing!

Hiding UX stuff is considered an anti-pattern and should be avoided!

Reply:
I think we could move this button to the other timelapse settings. Most of the time the "Render" button will be there anyway, and for cases where it isn't I think having it in the usual settings seems fine. What do you think?

Reply:
Makes sense to me!

Reply:
Fixed in https://github.com/fluidd-core/fluidd/pull/610/commits/9decaabdffec0a24d9e5fbbf5f65c3eceabecc63 😉